packages/block-library/src/social-links/style.scss

```diff
@@ -6,6 +6,11 @@
 	// Some themes give all <ul> default margin instead of padding.
 	margin-left: 0;
 
+	// Unset background colors that can be inherited from Global Styles with extra specificity.
+	&.wp-block-social-links {
+		background: none;
```

Comment:
Can we lower this to be just a single `.wp-block-social-links` selector like the rules above? I don't think that it makes a big difference tbh but it's nice in case anyone does want to target it directly.

Reply:
I'm fairly sure that when I tried, that didn't have enough specificity. Let me just verify that.

Reply:
On investigation, _in the editor_ it gets specificity to override this rule:
```
.editor-styles-wrapper ol, .editor-styles-wrapper ul {
    background-color: var(--wp--preset--color--vivid-green-cyan);
}
```

On the frontend, it need only override this rule:

```
ol, ul {
    background-color: var(--wp--preset--color--vivid-green-cyan);
}
```

I can change it so we have a low-specificity reset in style.scss, and higher specificity in editor.scss, sound good?

As a separate or alternative, we could look at how to reduce how editor styles are output, perhaps _`:where(.editor-styles-wrapper) ul`, etc.?